src/db/db.c

```diff
@@ -100,35 +100,45 @@ dbNew(PgClient *client, ProtocolClient *remoteClient, const Storage *const stora
 /***********************************************************************************************************************************
 Execute a query
 ***********************************************************************************************************************************/
-static VariantList *
-dbQuery(Db *this, const String *query)
+static Pack *
+dbQuery(Db *this, const PgClientQueryResult resultType, const String *const query)
 {
     FUNCTION_LOG_BEGIN(logLevelDebug);
         FUNCTION_LOG_PARAM(DB, this);
+        FUNCTION_LOG_PARAM(STRING_ID, resultType);
         FUNCTION_LOG_PARAM(STRING, query);
     FUNCTION_LOG_END();
 
     ASSERT(this != NULL);
     ASSERT(query != NULL);
 
-    VariantList *result = NULL;
+    Pack *result = NULL;
```

Comment:
should we assert that resultType is not null at line 113

Reply:
It's not a pointer, but we can at least make sure it is not zero. Done at 5822d3c3e5.